tools/ocean/argo_decoder.xml

```diff
@@ -0,0 +1,76 @@
+<tool id="coriolis_data_processing_chain_for_argo_floats" name="Coriolis data processing" version="@VERSION@" profile="22.05">
+    <description>chain for argo floats</description>
+    <macros>
+        <token name="@VERSION@">061a</token>
+        <token name="@VERSION_DOCKER@">@VERSION@-galaxy</token>
+    </macros>
+    <requirements>
+        <container type="docker">ghcr.io/euroargodev/coriolis-data-processing-chain-for-argo-floats:@VERSION_DOCKER@</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        
+        ## prepare outputs directory
+        mkdir -p /mnt/data/output/log &&
+        chmod -R 770 /mnt/data/output &&
+
+        ## load inputs into data directory
+        cp '$raw_data' '/tmp/raw_data.zip' &&
+        unzip /tmp/raw_data.zip -d /mnt/data &&
+
+        ## execute decoder script
+        \$APP_HOME/\$APP_RUN_FILE /mnt/runtime 'rsynclog' 'all' 'configfile' '/app/config/_argo_decoder_conf_ir_sbd.json' 'xmlreport' 'argo-decoder-report.xml' 'floatwmo' '$float_wmo' 'PROCESS_REMAINING_BUFFERS' '1' &&
+
+        ## copy (because we need to be in working directory) and create output archive
+        cd /mnt/data/output &&
+        zip -r argo-decoder-outputs.zip . &&
+        cp argo-decoder-outputs.zip '$outputs_data'
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="raw_data" type="data" format="zip" label="Archived input argo raw data"/>
+        <param name="float_wmo" type="integer" label="Input WMO number" help="Input float WMO numer (example : 6903014)"/>
+    </inputs>
+    <outputs>
+        <data name="outputs_data" format="zip" from_work_dir="argo-decoder-outputs.zip" label="argo decoder netcdf outputs"/>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="1">
+            <param name="raw_data" value="data_input_20240319.zip"/>
+            <param name="float_wmo" value="6903014"/>
+            <output name="outputs_data" value="argo-decoder-outputs.zip"/>
```

Comment:
what do you think about using https://docs.galaxyproject.org/en/latest/dev/schema.html#assertions-for-archives-zip-tar

To assert certain files in this archive, this would also enable us to remove this file.